.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
+language: java
+jdk:
+  - oraclejdk8
+services:
+  - docker
+before_install:
+  - sudo apt-get -qq update
```

Comment:
Ah ok i didnt know that,  i just thought that -qq might look less clear then -q=2 but its redundant if -q=2 does not work